sda-commons-dependencies/build.gradle

```diff
@@ -200,7 +200,7 @@ dependencies {
     // Kafka
     api "org.apache.kafka:kafka_2.13:$kafkaVersion"
     api "org.apache.kafka:kafka-clients:$kafkaVersion"
-    api "org.apache.zookeeper:zookeeper:3.6.4" // same as https://central.sonatype.com/artifact/org.apache.kafka/kafka_2.13/3.5.0
+    api "org.apache.zookeeper:zookeeper:3.8.1" // same version as Kafka Client 3.6.0
```

Comment:
As far as I see in the notes - [here](https://downloads.apache.org/kafka/3.6.0/RELEASE_NOTES.html):
> [[KAFKA-14661](https://issues.apache.org/jira/browse/KAFKA-14661)] - Upgrade Zookeeper to 3.8.2
and respective [PR](https://github.com/apache/kafka/pull/13260/files#diff-a74b4a65ab1f4aed61344787bb9654ba7e835154730653848c88ca11f9965dc0R264-R265) they updated ZK to `3.8.2` 🤔

Reply:
Updated to 3.8.2. I guess there is a small mistake in the [documentation](https://kafka.apache.org/36/documentation.html#upgrade_360_notable).